microcosm_postgres/store.py

```diff
@@ -167,6 +164,12 @@ def search(self, *criterion, **kwargs):
         query = self._filter(query, **kwargs)
         return query.all()
 
+    def expunge(self, instance):
+        return self.session.expunge(instance)
+
+    def merge(self, instance, new_instance):
```

Comment:
I need these for some other work I'm doing with polymorphic types. It turns out that if you `PATCH` a polymorphic object but don't specify the column it's polymorphic *on*, things fail.